tools/edgeDeviceFiles.sh

```diff
@@ -287,7 +288,7 @@ function gatherHorizonFiles () {
 function pullAgentInstallScript () {
 	echo "Pulling agent-install.sh script..."
 
-	curl -O https://raw.githubusercontent.com/open-horizon/anax/master/agent-install/agent-install.sh && \
+	curl -O https://raw.githubusercontent.com/open-horizon/anax/v4.0/agent-install/agent-install.sh && \
```

Comment:
Shouldn't this change only be put in the `edgeDeviceFiles.sh` in the `v4.0` branch of the examples repo? I.e. the v4.0 version of `edgeDeviceFiles.sh` gets the v4.0 version of `agent-install.sh` and the master version of `edgeDeviceFiles.sh` gets the master version of `agent-install.sh`?

Reply:
changed the base of the PR to the `v4.0` branch